### PR TITLE
Create initial event protocol

### PR DESCRIPTION
**Pull request by zachlatta**
This change adds an initial version of our event protocol that includes what we learned from running The Hack Series.

### CODE REVIEW

File: event_protocol.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Event Protocol
+
+This document contains our checklist that must be completed for all events run
+directly by Hack Club.
+
+## Preparation
+
+- [ ] Link to our [Code of Conduct][code_of_conduct] in marketing materials
+
+[code_of_conduct]: https://github.com/hackclub/hackclub/blob/master/CONDUCT.md
```
Comment (paked) [inline event_protocol.md]: I wish we could do this relatively

Comment (zachlatta) [inline event_protocol.md]: Me too :-(.

I'm thinking about setting up something like https://gh.hackclub.io that we can use to link to files in our `hackclub/hackclub` repo. An example would be https://gh.hackclub.io/CONDUCT.md.

It's not perfect, but I think it's better than having all of that `github.com/.../...` cruft in our URLs.

Comment (paked) [inline event_protocol.md]: @zachlatta i like it

**Comment by paked** [comment]
:<3:

**Pull request merged**